src/component/homepage/Card.jsx

```diff
@@ -1,9 +1,45 @@
-import React from 'react'
+import React from "react";
 
-function Card() {
+function Card({ article }) {
```

Comment:
This component is displaying a list of articles so the name Card for this component seems very confusing